docs/source/user-guide/general/drift_guide.rst

```diff
@@ -0,0 +1,186 @@
+However, we may need to act differently on it.
+For example, a fraud prevention company predicts fraudulent payments in 80% accuracy. Suddenly, the company notices that
+that they are getting more fraudulent payments recently. They are still predicted in 80% accuracy, but those 20% they
+get wrong are now more costly, and therefore the company needs to adjust its actions - perhaps adjusting the model
+decision threshold, or adding a manual review process.
+
+For more on the different types of drift, `see here <https://deepchecks.com/data-drift-vs-concept-drift-what-are-the-main-differences/>`_
+
+
+What Are the Causes of Drift?
+==============================
+
+(Is this interesting on its own? Or should this be a part of the description of each drift?)
+
+
+How Do You Detect Drift?
+=========================
+
+In general, drift detection is done by comparing the newer and older data, and seeing whether they are derived from
+the same underlying distribution. This is actually not a "yes or no" question - Instead, we ask "what is the probability
+that these 2 distributions are the same?".
+
+There are many methods to detect drift. Here, we will elaborate on 2:
+
+Detection by Statistical Test
+-----------------------------
+This is the simplest and most common drift detection method.
+This is done by sampling a single variable (that can either be a data feature, the label or even the prediction) and
+running a statistical test that aims to measure the difference the 2 samples.
+There are many tests that can do this, such as the Kolmogorov-Smirnov test, Jensen-Shannon Divergence and more.
+In deepchecks, we found that the best results are given by:
+
+* For continuous numeric distributions - `Wasserstein metric (Earth Movers Distance) <https://en.wikipedia.org/wiki/Wasserstein_metric>`__
+* For discrete or categorical distributions - `Population Stability Index (PSI) <https://www.lexjansen.com/wuss/2017/47_Final_Paper_PDF.pdf>`__ or `Cramer's V <https://en.wikipedia.org/wiki/Cram%C3%A9r%27s_V>`__
+
+These methods have the advantage of being simple to use and produce explainable results. However, they are limited by
+checking each feature one at a time, and cannot detect drift that occurs over multiple variables (or they detect it
+multiple times)
+
+Detection by Domain Classifier
+------------------------------
+
+This is a method to detect multivariate drift, meaning that it can run on several variables, and even on the whole dataset.
+This is done by training a model to classify if a sample came from the train dataset or the new (test or production) dataset.
+If the classifier can easily predict which sample is from which dataset, it would mean that there are significant differences between these datasets.
+
+The main advantage of this method is that it can also uncover covariate drift, meaning drift in the data that does not
+affect the distribution of each individual variable, but does affect the relationship between them. For example, ?
```

Comment:
For example, you're predicting the income of a person from his city and education. Let's say a tech giant now moved into city A. This means that:
1. Given that a person lives in city A, he's more likely to have a more advanced degree (educated people moved to city A) - this is multivariate drift.
2. Given his education, a resident of city A now earns more. - this is concept drift.